packages/button/test/button.test.ts

```diff
@@ -304,6 +304,20 @@ describe('Button', () => {
         expect(clickSpy.callCount).to.equal(1);
         clickSpy.resetHistory();
 
+        el.dispatchEvent(
+            new KeyboardEvent('keypress', {
```

Comment:
![image](https://user-images.githubusercontent.com/13471233/134038287-c192e3da-3fb6-4f8b-96d8-9c6c05736e55.png)
 `key` for both `Enter` and `NumpadEnter` is `Enter`. IMO it should be used.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#properties

Reply:
Thanks for pointing that out @OpportunityLiu!